mindmeld/components/nlp.py

```diff
@@ -639,6 +639,18 @@ def extract_allowed_intents(self, allowed_intents):
 
         return nlp_components
 
+    @staticmethod
+    def print_inspect_stats(stats):
+        """
+        Prints formatted output matrix
```

Comment:
If this is a public method we should provide some documentation on the stats argument (what is its format, etc)

Reply:
if stats is an empty [], what will the output to str be?

Reply:
It would just print two new lines.